### PR TITLE
Ben smyth comments

### PR DESCRIPTION
**Pull request by russhousley**
Resolve comments from Ben Smyth.

### CODE REVIEW

File: draft-ietf-tls-external-psk-guidance.md [hunk 1]
```diff
@@ -106,34 +107,36 @@ informative:
...
 # Introduction
 
-There are many resources that provide guidance for password generation and verification aimed towards improving security.
-However, there is no such equivalent for external Pre-Shared Keys (PSKs) in TLS. This document aims to reduce
-that gap. It lists TLS security properties provided by PSKs under certain assumptions and demonstrates
-how violations of these assumptions lead to attacks. This document also discusses PSK use
-cases, provisioning processes, and TLS stack implementation support in the context of these
-assumptions. It provides advice for applications in various use cases to help meet these
-assumptions.
+This document provides guidance on the use of external Pre-Shared Keys (PSKs)
```
Comment (chris-wood) [inline draft-ietf-tls-external-psk-guidance.md:123]: Can we revert this particular change? (cc @cbartle891)

File: draft-ietf-tls-external-psk-guidance.md [hunk 2]
```diff
@@ -150,17 +153,20 @@ External PSK authentication in TLS allows endpoints to authenticate connections
 using previously established keys. These keys do not provide protection
 of endpoint identities (see {{endpoint-privacy}}), nor do they provide
 non-repudiation (one endpoint in a connection can deny the conversation).
+Protection of endpoint identities and protection against an endpoint denying
+the conversation are possible when a fresh TLS handshake is performed.
```
Comment (chris-wood) [inline draft-ietf-tls-external-psk-guidance.md:162]: What is the point of this new text?

**Review by chris-wood — CHANGES_REQUESTED** [review]

File: draft-ietf-tls-external-psk-guidance.md (hunk 1 above)
Comment (russhousley) [reply draft-ietf-tls-external-psk-guidance.md:123]: This addresses a comment.  It you want to keep it, it belongs later in the Introduction.

Comment (chris-wood) [reply draft-ietf-tls-external-psk-guidance.md:123]: That's fine then. Can you please move it to some place later in the intro? (I just don't want to lose it.)

File: draft-ietf-tls-external-psk-guidance.md (hunk 2 above)
Comment (russhousley) [reply draft-ietf-tls-external-psk-guidance.md:162]: Look at the two sentences that come before this one.  Those sentences say that the PSK does not offer these things, so if you need these things, a fresh handshake is the way to get them.

File: draft-ietf-tls-external-psk-guidance.md (hunk 1 above)
Comment (russhousley) [reply draft-ietf-tls-external-psk-guidance.md:123]: I put it in the second paragraph in the intro.

File: draft-ietf-tls-external-psk-guidance.md (hunk 2 above)
Comment (chris-wood) [reply draft-ietf-tls-external-psk-guidance.md:162]: Yes, I get that, but that seems obvious. Does it need to be said explicitly?

Comment (chris-wood) [reply draft-ietf-tls-external-psk-guidance.md:162]: (I do not feel strongly about this, so let's just land it.)

**Review by chris-wood — APPROVED** [review]

**Comment by chris-wood** [comment]
@russhousley can you please cut a new version of the draft?

**Comment by russhousley** [comment]
Done.  Revision -02 has been submitted.

**Comment by russhousley** [comment]
I am getting a permission error when I try to set the tag for "draft-ietf-tls-external-psk-guidance-02".  Everything else is done, and I have used my local file to submit the Internet-Draft.  Please set the tag when you have a minute.

**Pull request merged**